plugins/languages/nvim-orgmode.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "orgmode";
+  originalName = "nvim-orgmode"; 
+  defaultPackage = pkgs.vimPlugins.orgmode; 
+
+  maintainers = [ helpers.maintainers.GaspardMonge ]; 
```

Comment:
This should be your maintainer entry you added to lib/maintainers. That user was just an example showing accessing maintainers list from the helpers scope if you're not in nixpkgs maintainer list.